src/Microsoft.DotNet.XHarness.CLI/Commands/XHarnessVersionCommand.cs

```diff
@@ -12,14 +13,21 @@ internal class XHarnessVersionCommand : Command
 
     public override int Invoke(IEnumerable<string> arguments)
     {
-        // Print the name of the tool and the version number unix style
-        // Example:
-        // Apple clang version 11.0.3 (clang-1103.0.32.29)
-        // Target: x86_64-apple-darwin19.4.0
-        // InstalledDir: /Applications/Xcode114.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
         var version = GetAssemblyVersion();
-        Console.WriteLine($"XHarness version {version.ProductVersion} ({version.OriginalFilename})");
-        Console.WriteLine($"InstalledDir: {version.FileName}");
+
+        if (arguments.Contains("--full") || arguments.Contains("-f") || arguments.Contains("-v"))
+        {
+            // Print the name of the tool and the version number unix style
+            // Example:
+            // Apple clang version 11.0.3 (clang-1103.0.32.29)
+            // Target: x86_64-apple-darwin19.4.0
+            // InstalledDir: /Applications/Xcode114.app/Contents/Developer/Toolchains/XcodeDefault.xctoolchain/usr/bin
+            Console.WriteLine($"XHarness version {version.ProductVersion} ({version.OriginalFilename})");
+            Console.WriteLine($"InstalledDir: {version.FileName}");
+        }
+
+        Console.WriteLine(version.ProductVersion);
```

Comment:
if Line :18 `true` we'll print `ProductVersion` 2 times: line :29, line :25

Reply:
Right, I'm stupid. Thanks!